content/docs/rfcs/14/README.md

```diff
@@ -1,120 +1,102 @@
 ---
 slug: 14
 title: 14/WAKU2-MESSAGE
-name: Waku v2 Message
-status: draft
+name: Waku v2 message RFC
+status: stable
+category: Standards Track
```

Comment:
I am not sure if I picked the correct category for this RFC. Is this the right RFC?

Reply:
[Here](https://github.com/vacp2p/rfc/blob/master/content/docs/rfcs/template/README.md) you can find the template, but I cannot find the different categories description

Reply:
think this is correct but defer to @kaiserd